### PR TITLE
chore(docs): fixing wrong desciption for `pg.worker.count`

### PR DESCRIPTION
**Pull request by amyshwang**

### CODE REVIEW

File: docs/reference/configuration.md [hunk 1]
```diff
@@ -442,7 +442,7 @@ PostgresSQL wire protocol.
...
 | pg.date.locale                   | en           | The locale to handle date types.                                                                                                                                                                  |
 | pg.timestamp.locale              | en           | The locale to handle timestamp types.                                                                                                                                                             |
-| pg.worker.count                  | 0            | Number of dedicated worker threads assigned to write data. When `0`, the writer jobs will use the shared pool.                                                                                    |
+| pg.worker.count                  | 0            | Number of dedicated worker threads assigned to handle PGWire queries. When `0`, the writer jobs will use the shared pool.                                                                                    |
```
Comment (puzpuzpuz) [inline docs/reference/configuration.md:445]:
```suggestion
| pg.worker.count                  | 0            | Number of dedicated worker threads assigned to handle PG Wire queries. When `0`, the jobs will use the shared pool.                                                                                    |
```

Comment (amyshwang) [reply docs/reference/configuration.md:445]: We use PGWire - https://questdb.io/blog/2022/05/09/questdb-release-6-3/#read-only-mode-for-pgwire-protocol
Let me confirm the usage.

Comment (amyshwang) [reply docs/reference/configuration.md]: Brief discussion with @pswu11 --> I am going to use PGWire; change the other case on this page. I have deleted "writer" in line 445.

Comment (puzpuzpuz) [reply docs/reference/configuration.md]: Could you also update Prometheus page in this PR?

Comment (amyshwang) [reply docs/reference/configuration.md]: Piggy-bag to https://github.com/questdb/questdb.io/pull/1067 :)

**Review by puzpuzpuz — APPROVED** [review]

**Pull request merged**